cldfbench_eurasianinventories.py

```diff
@@ -0,0 +1,296 @@
+# **NOTE**: While this script is named `cldfbench_eurasianinventories.py`,
+#           it is not integrated with `cldfbench` yet; the name is only
+#           here as a placeholder for future integration.
+
+from collections import OrderedDict
+from pathlib import Path
+import base64
+import csv
+import hashlib
+import json
+import logging
+
+# Import MPI-SHH libraries
+from clldutils.path import git_describe
+from pycldf import StructureDataset
+from pyglottolog import Glottolog
+import pyclts
+
+# BIPA features for descriptors
+BIPA_FEATURES = [
```

Comment:
As I mentioned before: bipa features can be derived from clts directly, non need to add it here.

Reply:
@tresoldi  See https://github.com/cldf-clts/pyclts/issues/16 - and let me know whether you think the backwards compatible solution if better or not necessary.

Reply:
@xrotwang perfect and elegant. :smile: